kdwsdl2cpp/src/main.cpp

```diff
@@ -290,8 +293,38 @@ int main(int argc, char **argv)
     KWSDL::Compiler compiler;
 #if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
     if (!pkcs12File.isEmpty()) {
-        if (!Settings::self()->loadCertificate(pkcs12File, pkcs12Password))
-            return -1;
+        QFile certFile(pkcs12File);
+        if (certFile.open(QFile::ReadOnly)) {
+            QSslKey key;
+            QSslCertificate certificate;
+            QList<QSslCertificate> caCertificates;
+            auto mCertificateLoaded = QSslCertificate::importPkcs12(&certFile,
```

Comment:
The naming needs to be adapted (no 'm' prefix) --> const bool certificateLoaded =

Reply:
Fixed, thanks for spotting it!